code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -40,6 +40,25 @@
 		overlay_x = 15, \
 		overlay_y = 10)
 
+/obj/item/gun/energy/disabler/smg
+	name = "disabler smg"
+	desc = "An automatic disabler variant, as opposed to the conventional model, boasts a higher ammunition capacity at the cost of slightly reduced beam effectiveness."
+	icon_state = "disabler_smg"
+	ammo_type = list(/obj/item/ammo_casing/energy/disabler/smg)
+	shaded_charge = 1
+	var/allow_akimbo = FALSE
```

Comment:
Any reason this is a var? It doesn't have any subtypes and an admin changing it at runtime won't do anything.

Reply:
> Any reason this is a var? It doesn't have any subtypes and an admin changing it at runtime won't do anything.

Laser carbine does the same, thought it makes sense.

Reply:
Hi, person who designed laser carbines here. It doesn't technically have to be a var, you could just pass `FALSE` to the component directly. I just thought making it explicit would be better.